### PR TITLE
在table组件中添加Customrender功能一样的unitComponent但是不受编辑影响，同时添加上传组件

### PR DESCRIPTION
**Pull request by Master-Jian**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [x] Pull request template structure not broken

### `Type`

> ℹ️ What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### `Checklist`

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

> 👉 _Put an `x` in the boxes that apply._

- [x] My code follows the style guidelines of this project
- [x] Is the code format correct
- [x] Is the git submission information standard?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/views/demo/table/EditRowTable.vue [hunk 1]
```diff
@@ -78,6 +80,19 @@
         },
       ],
     },
+    {
+      title: '图片',
+      dataIndex: 'imgs',
+      width: 200,
+      unitComponent: ({ text }) => h(Image, { src: text, width: 70 }), // 显示自定义组件
```
Comment (wangjue666) [inline src/views/demo/table/EditRowTable.vue:87]: 我觉得不需要再增加`unitComponent`字段  已经有 `customRender` 这个字段来控制自定义渲染了

```
{
      title: '图片',
      dataIndex: 'imgs',
      width: 200,
      customRender: ({ record }) => h(Image, { src: record.imgs, width: 70 }), // 显示自定义组件
      editRow: false,
}
```

只需要改变下edit那块的渲染逻辑就好， 你认为呢

**Review by wangjue666 — CHANGES_REQUESTED** [review]

File: src/views/demo/table/EditRowTable.vue (hunk 1 above)
Comment (Master-Jian) [reply src/views/demo/table/EditRowTable.vue:87]: 确实，如果能控制edit逻辑的话，不需要此字段来控制。那如何改动edit这块逻辑

Comment (wangjue666) [reply src/views/demo/table/EditRowTable.vue:87]: 害，有个 `editRender` 字段， 这样写就没问题


```
{
      title: '图片',
      dataIndex: 'imgs',
      width: 200,
      editRender: ({ record }) => h(Image, { src: record.imgs, width: 70 }), // 显示自定义组件
      editRow: true,
      editComponent: 'ImageUpload',
      editComponentProps: {
        api: uploadApi,
        accept: ['png', 'jpeg', 'jpg'],
        maxNumber: 1,
      },
    },
```

Comment (wangjue666) [reply src/views/demo/table/EditRowTable.vue:87]: editrender 这个字段名字起的不好 乍一看还以为是可编辑情况下的自定义渲染呢

**Comment by BobinYang** [comment]
BasicTable在可编辑情况下如何自定义渲染呢？

**Comment by wangjue666** [comment]
> BasicTable在可编辑情况下如何自定义渲染呢？

暂时只支持editcomponent, 太麻烦的编辑组件你或许应该考虑弄成弹窗+form表单

File: src/views/demo/table/EditRowTable.vue (hunk 1 above)
Comment (Master-Jian) [reply src/views/demo/table/EditRowTable.vue:87]: 确实这名字起的不是很好，被忽略了，而且没有特别明显的标志所以我才加了这个。回头我看下

**Pull request closed without merging**